features/support/env.rb

```diff
@@ -150,5 +150,15 @@ def extract_process_id(data)
   end
 
   expect(binaries.size).to eq(1)
-  expect(File.executable?(binaries[0])).to eq(true)
+  expect(File.executable?(binaries.first)).to eq(true)
+end
+
+Then(/^the binary "([^"]*)" should be installed with version "([^"]*)"$/) do |binary, version|
+  binaries = []
+  Find.find("#{ENV['VAGRANT_HOME']}/data/service-manager/bin") do |path|
```

Comment:
I don't know but `#{ENV['VAGRANT_HOME']}/data/service-manager/custom-path"` is not working in command

`bundle exec vagrant service-manager install-cli docker --path #{ENV['VAGRANT_HOME']}/data/service-manager/docker`